e2e/nomostest/clean.go

```diff
@@ -68,6 +70,11 @@ func Clean(nt *NT) error {
 		nt.T.Logf("[CLEANUP] Test environment cleanup took %v", elapsed)
 	}()
 
+	if err := uninstallAllPackages(nt); err != nil {
```

Comment:
I think this is going to fail whenever Clean is called [here](https://github.com/GoogleContainerTools/kpt-config-sync/blob/main/e2e/nomostest/new.go#L351), because the reconciler-manager Deployment will be not found. This will impact periodics because they set `--create-clusters=lazy`

Reply:
Pulled out an `isReconcilerManagerHealthy` func to check that first without failing the test when it's unhealthy/uninstalled.